spec/features/notification_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe 'Notification' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.